relay/submit.go

```diff
@@ -98,6 +98,25 @@ func (rs *Relay) SubmitBlock(ctx context.Context, m *structs.MetricGroup, uc str
 		return err
 	}
 
+	// TODO
```

Comment:
; > ?

Reply:
TODO is because we have to enable this in the future.